autoload/test/viml/themis.vim

```diff
@@ -0,0 +1,38 @@
+if !exists('g:test#viml#themis#file_pattern')
+  let g:test#viml#themis#file_pattern = '.vim$'
+endif
+
+if !exists('g:test#viml#themis#executable')
+  let g:test#viml#themis#executable = 'themis'
+endif
+
+function! test#viml#themis#test_file(file) abort
+  return a:file =~# g:test#viml#themis#file_pattern
+        \ && !empty(filter(readfile(a:file), 'v:val =~# ''\<themis#suite\s*('''))
+endfunction
+
+function! test#viml#themis#build_position(type, position) abort
+  if a:type == 'nearest' || a:type == 'file'
+    return [a:position['file']]
+  else
+    return []
+  endif
+endfunction
+
+function! test#viml#themis#build_args(args) abort
+  let args = a:args
+
+  if exists('test#viml#themis#args')
+    let args = g:test#viml#themis#args + args
+  endif
```

Comment:
This shouldn't be necessary, because you can already configure CLI arguments for any runner via `g:test#<language>#<runner>#options`, see the [docs](https://github.com/janko-m/vim-test#cli-options).

Reply:
Removing this makes this function just return `a:args` unchanged, do we need to define the function at all in this case?

Reply:
At the moment we still need to define it even if it just has `return a:args`.